lib/Dialect/FIRRTL/FIRRTLFolds.cpp

```diff
@@ -1267,18 +1267,22 @@ LogicalResult SubaccessOp::canonicalize(SubaccessOp op,
 /// exactly one connect that sets the value as its destination.  This returns
 /// the operation if found and if all the other users are "reads" from the
 /// value.
-static bool isOnlyConnectToValue(ConnectOp connect, Value value) {
+static ConnectOp getSingleConnectUserOf(Value value) {
```

Comment:
Nice.  This is probably worth promoting up to FIRRTLOps.cpp/h given that other firrtl transforms often need a similar thing.